schema.dbml

```diff
@@ -510,6 +511,79 @@ Enum assay_enum {
   "MLV pseudovirus"
 }
 
+Table assay as AS {
+  ref_name varchar
+  assay_name assay_enum
+  upper_titer_limit int
+  lower_titer_limit int
+}
+
+Table fold as FD {
```

Comment:
How about change the name to rx\_fold. Let's try avoiding using table names that are same as column names. We can also add the potency\_type to the table.